recipes/gptcache/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "gptcache" %}
+{% set version = "0.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gptcache-{{ version }}.tar.gz
+  sha256: d4a18a7587e45916fb406c2e949f2d225cdf1f8fecd0f68ee3de2e7a9cfe30c2
+
+build:
+  skip: true  # [win or py<=38]
```

Comment:
I assume this due to missing dependencies on Windows? If so, this recipe can be made `noarch: python` and we can ignore the CI failures on Windows.

Reply:
Yes, that what the reason.
Makes sense, if we want to still ship it as `noarch` (which it is, right? it's just missing some dependencies on windows)
I just didn't know the conda-forge guidelines there.